lib/ts/recipe/accountlinking/types.ts

```diff
@@ -107,7 +107,7 @@ export type RecipeInterface = {
         | {
```

Comment:
4 poossible outputs:
- success -> OK
- it's already a primarty user id -> OK + boolean
- it's linked to some primary user -> RECIPE_USER_ID_ALREADY_LINKED_WITH_PRIMARY_USER_ID_ERROR
- email already belongs to a primary user -> ACCOUNT_INFO_ALREADY_ASSOCIATED_WITH_ANOTHER_PRIMARY_USER_ID_ERROR

Reply:
This also has downstream affect in our apis where in we check for if the result is OK || (if it's RECIPE_USER_ID_ALREADY_LINKED_WITH_PRIMARY_USER_ID_ERROR && primaryUserId == recipeUserId). These will just need to be if it's OK now